libs/composite-product/driver/magento/src/fragments/bundled-product.ts

```diff
@@ -0,0 +1,42 @@
+import { gql } from 'apollo-angular';
+
+export const magentoBundledProductFragment = gql`
+  fragment magentoBundledProduct on BundleProduct {
+		items {
+			option_id
+			position
```

Comment:
`position` (and the one in the option) doesn't seem like they are ever used. If that's true then they should be removed.

Reply:
removed in this PR, then I'll rebase: https://github.com/graycoreio/daffodil/pull/1705

Reply:
rebased, and I made the same changes I made in the `@daffodil/product/driver/magento` package.